src/resonanceReconstruction/breitWigner/singleLevel/Type/test/Type.test.cpp

```diff
@@ -21,12 +23,19 @@ const std::vector< double >& elastic();
 const std::vector< double >& capture();
 
 SCENARIO( "Integration test" ){
+  auto start = std::chrono::high_resolution_clock::now();
```

Comment:
you're conflating the parsing step and reconstruction step by beginning your timer here. If you're interested in comparing to `csslbw`, you should move this to (what is now) line 31